src/rgw/rgw_s3select.cc

```diff
@@ -427,6 +427,9 @@ int RGWSelectObj_ObjStore_S3::run_s3select_on_csv(const char* query, const char*
     csv.use_header_info=true;
   }
 
+  if(m_outputFormat == OutputFormat::JSON) {
```

Comment:
Please fix spacing/indentation nits as mentioned in other PR.

Reply:
> Please fix spacing/indentation nits as mentioned in other PR.

remove the new-line on line:429?